ui/admin/app/components/form/host/aws/index.hbs

```diff
@@ -69,27 +88,33 @@
     </F.HelperText>
   </InfoField>
 
-  <form.fieldset>
-    <:title>{{t 'form.ip_addresses.label'}}</:title>
-    <:body>
-      {{#each @model.ip_addresses as |ip_address|}}
-        <form.input
+  <Hds::Form::Fieldset as |F|>
+    <F.Legend>{{t 'form.ip_addresses.label'}}</F.Legend>
+    <F.HelperText>{{t 'form.ip_addresses.help'}} </F.HelperText>
+    {{#each @model.ip_addresses as |ip_address|}}
+      <F.Control>
+        <Hds::Form::TextInput::Base
           @value={{ip_address}}
-          readonly={{true}}
-          @disabled={{true}}
+          disabled={{true}}
+          @width='unset'
         />
-      {{/each}}
-    </:body>
-  </form.fieldset>
+      </F.Control>
+    {{/each}}
+  </Hds::Form::Fieldset>
```

Comment:
This displays a bit differently now: it shows the border for the input fields for each of these without overriding HDS styles. I thought it actually looked fine and the behavior was a bit better than before. 

It looks like if there were too many fields, it would force them all on one line regardless and become unable to display them (though this is unlikely)
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/5783847/230143429-a815949e-e154-4ad4-b7c4-5b6a312f07e0.png">

Here's a normal scenario: 
<img width="1559" alt="image" src="https://user-images.githubusercontent.com/5783847/230143614-32a1cd1a-fa5f-41d1-877a-b582053b0352.png">

Let me know your thoughts

Reply:
I like the bordered inputs.